src/Italy/Mews.Fiscalizations.Italy/Communication/Dto/InvoiceStateResult.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using Newtonsoft.Json;
 
-namespace Mews.Fiscalizations.Uniwix.Communication.Dto
+namespace Mews.Fiscalizations.Italy.Communication.Dto
```

Comment:
This is still specific to Uniwix, so I think it should at least be in a separate namespace.